src/templates/work/programs/index.hbs

```diff
@@ -133,21 +146,19 @@
         <li class="slide-1">
           <div>
             <h1>Strong Young Womens Program</h1>
-            <h2>(SYWP)</h2>
             <p class="see-more">Click to see more</p>
           </div>
         </li>
-        <li class="slide-2">
+        <li class="slide-2 slide-text">
           <div>
             <h2>The Strong Young Women’s Program delivers gender-specific health messages to young women to a range of stakeholder groups in Alice Springs and remote Central Australian and Top End communities.</h2>
             <p>In addition to highlighting the link between lifestyle choices and well being over an individuals life course, the program raises awareness of modifiable risk behaviours with an emphasis on alcohol use and Fetal Alcohol Spectrum Disorder.</p>
           </div>
         </li>
-        <li class="slide-3">
+        <li class="slide-3 slide-map">
           <div>
-            <h3>SYWP</h3>
             <h1>Where We Deliver</h1>
-            <img src="/assets/img/work/programs/SYWP-map.png" />
+            <img src="/assets/img/work/programs/sywp-map.png" />
```

Comment:
Git does not pick up on changing file names from lowercase to uppercase. It may well be this has not been `git add`-ed and will be broken once I pull in or when we push it to staging. The change probably only exists on your local.